crates/core/src/repofile/snapshotfile.rs

```diff
@@ -1368,4 +1371,14 @@ mod tests {
         assert_eq!(crit.paths, is_path);
         assert_eq!(crit.tags, is_tags);
     }
+
+    #[rstest]
+    #[case(vec![], "")]
```

Comment:
This case panicked before.

Reply:
Yes, good catch and good to have it repaired!